test/selenium/OneOffContributionsSpec.scala

```diff
@@ -20,6 +20,41 @@ class OneOffContributionsSpec extends FeatureSpec with GivenWhenThen with Before
 
   feature("Sign up for a one-off contribution") {
 
+    scenario("One-off contribution sign-up with Stripe - GBP") {
+
+      val expectedPayment = 50.00
```

Comment:
I don't think we need 2 vals for expectedPayment

Reply:
Also it doesn't look like we're ever checking that expectedPayment matches the actual payment amount (as we do in the PayPal test here: https://github.com/guardian/support-frontend/pull/673/files#diff-8ce9cc1c1c2eecc8466a14b15efa9c32R91)

Reply:
this should be fixed in latest commit.